### PR TITLE
ui: fix for elements width inconsistency when modal is open

### PR DESCRIPTION
**Pull request by karolina-siemieniuk-morawska**
The issue was placing the modal outside DetailPageContainer <Row /> element. Moved JournalTitlesModal inside <Row />

### CODE REVIEW

File: ui/src/journals/components/JournalTitlesListModal.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const JournalTitlesListModal = ({
     visible={modalVisible}
     onCancel={onModalVisibilityChange}
     footer={null}
+    maskStyle={{ width: '100vw', overflow: 'hidden' }}
```
Comment (drjova) [inline ui/src/journals/components/JournalTitlesListModal.tsx:17]: I'm not sure this is the right solution, why this is happening? We have other modals and they work properly.

Comment (drjova) [reply ui/src/journals/components/JournalTitlesListModal.tsx:17]: @karolina-siemieniuk-morawska ☝️

**Comment by drjova** [comment]
@karolina-siemieniuk-morawska thanks! could you please attach a screenshot?

**Comment by karolina-siemieniuk-morawska** [comment]
@drjova Here's a recording 
https://user-images.githubusercontent.com/55505399/201104329-b21188ec-aa33-4528-a97c-cf55a68e438a.mov

**Comment by karolina-siemieniuk-morawska** [comment]
@drjova here's a recording:
Screen_Recording_2022-11-10_at_14_37_41_AdobeExpress

**Review by drjova — APPROVED** [review]

**Pull request merged**